### PR TITLE
Mar 2019 Cordova Electron 1.0.2 Release Blog Post

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected
none

### Motivation and Context
Release

### Description
Talk about Electron 1.0.2 Release

### Testing
None

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: www/_posts/2019-03-06-cordova-electron-release-1.0.1.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Cordova 9 (Not Released as of 2018/03/06)
+```
+cordova platform add electron@1.0.1
+```
+
+The original release was targeted for **Cordova CLI 9.x**. The decision to support on **Cordova CLI 8.x** was in the post-release phase. The **pre-Cordova 9** steps in the [previous release post](https://cordova.apache.org/announcements/2019/02/28/cordova-electron-release-1.0.0.html) were meant to be for this release.
+
+This release contains the changes to allow Electron to work correctly with **Cordova CLI 8.x**.
+
+Please note that all commands on **Cordova CLI 8.x** and that has the platform argument should use `cordova-electron`.
```
Comment (janpio) [inline www/_posts/2019-03-06-cordova-electron-release-1.0.1.md]: grammar

**Comment by janpio** [comment]
tbh the blog post would confuse me even more with the `cordova-electron`/`electron` stuff.

**Comment by erisu** [comment]
@janpio Check the latest push. 

I dropped all reference of `pre-Cordova 9.x` from the previous post, and this release post only mention `Cordova CLI 8.x`.

**Comment by janpio** [comment]
I don't know if this helps much. Those blog posts will be used forever by people on how to install the platform, so they both should probably contain the correct instructions that work.

**Comment by erisu** [comment]
They both work.

Cordova 9 can install with either `electron` or `cordova-electron` as the keyword.

But Cordova 8.x can only support `cordova-electron`. This is because Cordova Lib does not have the special alias wrapper for this npm package name.  All other platforms has aliases that maps our pre-define keyword to the npm package name. (For Example: `ios` => `cordova-ios`)

**Pull request merged**